lib/matplotlib/tests/test_ticker.py

```diff
@@ -191,6 +191,70 @@ def test_additional(self, lim, ref):
 
         assert_almost_equal(ax.yaxis.get_ticklocs(minor=True), ref)
 
+    def test_ndivs_rcParams_auto(self):
+        # when no parameter is given rcParams should be used
+        # auto
+        mpl.rc_context(rc={'xtick.minor.ndivs': 'auto'})
+        mpl.rc_context(rc={'ytick.minor.ndivs': 'auto'})
```

Comment:
This needs to be used as a context manager, or it has no effect. Or you can set `rcParams` directly; they'll be reset at the end of the test.